src/main/java/blackjack/domain/player/Gamers.java

```diff
@@ -13,29 +13,20 @@
 
 public class Gamers {
 
-    private final List<Player> gamers;
+    private final List<? extends Player> gamers;
 
-    public Gamers(List<String> names) {
-        validateDuplicate(names);
-        this.gamers = namesToGamers(names);
-    }
+    public Gamers(List<Gamer> gamers) {
+        validateDuplicate(gamers);
 
-    public Gamers(String... input) {
-        this(Arrays.asList(input));
+        this.gamers = gamers;
     }
 
-    private void validateDuplicate(List<String> names) {
-        if (names.size() != names.stream().distinct().count()) {
+    private void validateDuplicate(List<Gamer> gamers) {
+        if (gamers.size() != gamers.stream().map(Gamer::getName).distinct().count()) {
```

Comment:
메소드체이닝이 여러번 진행되는 스트림을 바로쓰기보단 변수로 분리해주는게 가독성이 더 좋을 것 같아요 :) 
```suggestion
        int gamerNameDistinctCount = gamers.stream()
                .map(Gamer::getName)
                .distinct()
                .count()
        if (gamers.size() != gamerNameDistinctCount) {
```

Reply:
오... 한줄로 끝내는게 좋다는 강박관념(?)이 사라지네요..!!! 감사합니다!